.drone.yml

```diff
@@ -447,23 +447,13 @@ steps:
     commands:
       - git fetch --tags --force
 
-  - name: static-windows
-    pull: always
-    image: techknowlogick/xgo:go-1.14.x
-    commands:
-      - apt update && apt -y install curl
-      - curl -sL https://deb.nodesource.com/setup_12.x | bash - && apt -y install nodejs
-      - export PATH=$PATH:$GOPATH/bin
-      - make frontend generate release-windows
-    environment:
-      GOPROXY: https://goproxy.cn # proxy.golang.org is blocked in China, this proxy is not
-      TAGS: bindata sqlite sqlite_unlock_notify
-
   - name: static
     pull: always
     image: techknowlogick/xgo:go-1.15.x
     commands:
-      - make release-linux release-darwin release-copy release-compress release-sources release-docs release-check
+      - curl -sL https://deb.nodesource.com/setup_14.x | bash - && apt -y install nodejs
```

Comment:
There's one more case of `setup_12.x` in the file, would you mind updating that as well? I missed those in the node 14 pr.

Reply:
good catch. thanks :)